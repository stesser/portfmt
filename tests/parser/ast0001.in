PORTNAME=llvm90
.   for option a in CLANG COMPILER_RT EXTRAS LLD LLDB OPENMP
WRKSRC_${option:tl}_$a=	${WRKDIR}/${${option}_DISTFILES}
. endfor
FOO=meh

. include <bsd.port.mk>
