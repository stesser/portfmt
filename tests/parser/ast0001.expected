{ VARIABLE, line 1, .name = PORTNAME, .modifier = =, .words = { llvm90 } }
{ FOR, line 2, .indent = 3, .bindings = { option, a }, .words = { CLANG, COMPILER_RT, EXTRAS, LLD, LLDB, OPENMP } }
	{ VARIABLE, line 3, .name = WRKSRC_${option:tl}_$a, .modifier = =, .words = { ${WRKDIR}/${${option}_DISTFILES} } }
{ VARIABLE, line 5, .name = FOO, .modifier = =, .words = { meh } }
{ COMMENT, line 6, .comment =  }
{ INCLUDE/BMAKE, line 7, .indent = 1, .path = bsd.port.mk, .sys = 1, .loaded = 0 }
